core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeExportService.java

```diff
@@ -137,14 +138,20 @@ public Map<String, String> generateExportMap(EnrolleeExportData exportData,
      * e.g. the columns needed to represent the survey questions.
      */
     public List<ModuleFormatter> generateModuleInfos(ExportOptions exportOptions, UUID studyEnvironmentId, List<EnrolleeExportData> enrolleeExportData) {
-        List<ModuleFormatter> moduleFormatters = new ArrayList<>();
-        moduleFormatters.add(new EnrolleeFormatter(exportOptions));
-        moduleFormatters.add(new ParticipantUserFormatter(exportOptions));
-        moduleFormatters.add(new ProfileFormatter(exportOptions));
-        moduleFormatters.add(new KitRequestFormatter());
-        moduleFormatters.add(new EnrolleeRelationFormatter());
-        moduleFormatters.add(new FamilyFormatter());
-        moduleFormatters.addAll(generateSurveyModules(exportOptions, studyEnvironmentId, enrolleeExportData));
+        List<ModuleFormatter> allSimpleFormatters = List.of(
+                new EnrolleeFormatter(exportOptions),
+                new ParticipantUserFormatter(exportOptions),
+                new ProfileFormatter(exportOptions),
+                new KitRequestFormatter(),
+                new EnrolleeRelationFormatter(),
+                new FamilyFormatter());
+
+        List<ModuleFormatter> moduleFormatters = allSimpleFormatters.stream().filter(
+                moduleFormatter -> !exportOptions.getExcludeModules().contains(moduleFormatter.getModuleName())
```

Comment:
When I open the data export page on demo, I'm getting this error:

```
java.lang.NullPointerException: Cannot invoke "java.util.List.contains(Object)" because the return value of "bio.terra.pearl.core.service.export.ExportOptions.getExcludeModules()" is null
	at bio.terra.pearl.core.service.export.EnrolleeExportService.lambda$generateModuleInfos$0(EnrolleeExportService.java:150)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:722)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at bio.terra.pearl.core.service.export.EnrolleeExportService.generateModuleInfos(EnrolleeExportService.java:151)
	at bio.terra.pearl.core.service.export.EnrolleeExportService.export(EnrolleeExportService.java:88)
	at bio.terra.pearl.api.admin.service.EnrolleeExportExtService.export(EnrolleeExportExtService.java:34)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
```